lib/analyzer/index.ts

```diff
@@ -1,75 +1,4 @@
-import * as Debug from "debug";
-import { DockerOptions } from "../docker";
-import * as dockerFile from "../docker-file";
-import * as apkAnalyzer from "./apk-analyzer";
-import * as aptAnalyzer from "./apt-analyzer";
-import * as binariesAnalyzer from "./binaries-analyzer";
-import * as imageInspector from "./image-inspector";
-import * as osReleaseDetector from "./os-release-detector";
-import * as rpmAnalyzer from "./rpm-analyzer";
+import { analyze as analyzeDynamically } from "./docker-analyzer";
```

Comment:
Maybe we can rename `docker-analyzer` to `dynamic-analyzer`?

Reply:
I think it would be rather new, right now everything in the dynamic part relates to Docker and I think it may be confusing if half of the things are called Docker and the other half "dynamic"... Maybe it would be better if "analyzeDynamically" got named as "analyzeWithDocker" here instead?

Reply:
Yep, makes sense 👍